### DISCUSSION TITLE
rendering unicode emoji in README.md not consistent with Github

### DISCUSSION THREAD

**Discussion by atao60** [npm/feedback#274] [Registry] [answered]
npmjs.com keeps the native unicode emoji in README.md, i.e. doing something like:
`<g-emoji class="g-emoji" alias="shield" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f6e1.png">🛡</g-emoji>`
When a native unicode is a fully qualified one, the look&feel stays close of Github one, i.e. with some colors. But with unqualified unicode, the emoji is displayed in black and white.

Not all emojis are concerned, only those for which exist both nonqualified and fully qualified unicodes. It's the case e.g. with 'shield' (fully qualified '1f6e1 fe0f' vs unqualified '1f6e1') or 'hammer_and_wrench' (fully qualified '1f6e0 fe0f' vs unqualified '1f6e0').

As some editors like VSC display all emojis in black and white, fully qualified or not, it's hard to spot wrong unicodes.

On Github there no such issue as it replaces all native emoji unicodes with png images, i.e. something like:
`<g-emoji class="g-emoji" alias="shield" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f6e1.png"><img class="emoji" alt="shield" src="https://github.githubassets.com/images/icons/emoji/unicode/1f6e1.png" width="20" height="20"></g-emoji>
`

 It should be great if npmjs.com were doing the same thing.

**Comment by MylesBorins** [comment, marked as answer]
Thanks for reporting @atao60 . I've created an internal ticket to track this bug. Could you share some example README.md files we could look at?

**Reply by atao60** [reply]
See https://github.com/atao60/fse-cli/blob/master/README.md with a unicode for emoji 'shield'.

The current version is with a full-qualified one. The result on NPM Registry is available here: https://www.npmjs.com/package/@atao60/fse-cli.

The commit where it has been changed is available here: https://github.com/atao60/fse-cli/commit/6655b5d3b15d6576404efde9b7c5fb720ddce9a4

A readme with the non qualified unicode emoji 'shield' is available here: https://www.npmjs.com/package/@atao60/fse-cli/v/0.0.54 (https://github.com/atao60/fse-cli/blob/262f0a49e384b8675af580d3fcb91fc79c0069af/README.md)

**Reply by MylesBorins** [reply]
@atao60 which browser are you on in which you are seeing the black/white shield. I am unable to reproduce

**Reply by atao60** [reply]
On the page https://www.npmjs.com/package/@atao60/fse-cli/v/0.0.54 with:

- Firefox86.0 (64 bits)
- Google Chrome Version 75.0.3770.142 (Build officiel) (64 bits)

both under Ubuntu 16 / 18 / 20

**Reply by MylesBorins** [reply]
This appears to be fixed, please lmk if you are still running into the issue
